exercises/high-scores/canonical-data.json

```diff
@@ -0,0 +1,91 @@
+{
+  "exercise": "high-scores",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Returns the list of scores",
+      "property": "scores",
+      "input": {
+        "scores": [3, 5, 2, 7]
+      },
+      "expected": [3, 5, 2, 7]
+    },
+    {
+      "description": "Returns the last score",
+      "property": "latest",
+      "input": {
+        "scores": [10, 0, 9, 3]
+      },
+      "expected": 3
+    },
+    {
+      "description": "Returns the highest score",
+      "property": "highest",
+      "input": {
+        "scores": [4, 10, 7]
+      },
+      "expected": 10
+    },
+    {
+      "description": "Returns the top 3 scores",
+      "cases": [
+        {
+          "description": "Returns personal bests",
+          "property": "top",
+          "input": {
+            "scores": [5, 3, 1]
+          },
+          "expected": [5, 3, 1]
+        },
+        {
+          "description": "Returns personal bests highest to lowest",
+          "property": "top",
+          "input": {
+            "scores": [2, 1, 3]
+          },
+          "expected": [3, 2, 1]
+        },
+        {
+          "description": "Returns personal bests when there are less than 3",
+          "property": "top",
+          "input": {
+            "scores": [3, 7]
+          },
+          "expected": [7, 3]
+        },
+        {
+          "description": "Returns personal bests when there is only one",
+          "property": "top",
+          "input": {
+            "scores": [4]
+          },
+          "expected": [4]
+        },
+        {
+          "description": "Returns three top scores from a long list",
+          "property": "top",
+          "input": {
+            "scores": [1, 3, 9, 3, 10, 2, 1, 0, 3, 4, 4, 7, 7]
+          },
+          "expected": [10, 9, 7]
+        }
+      ]
+    },
+    {
+      "description": "Returns the goal when highest is latest",
+      "property": "goal",
+      "input": {
+        "scores": [2, 4, 0, 3, 7]
+      },
+      "expected": "Your latest score was 7. That's your personal best!"
```

Comment:
Something about this bugs me. I think it is that this property returns a string while the others are arrays.

Reply:
Maybe if the property was something like `message` or `goal_report` that made it a bit clearer that we will be expecting a string rather than a numeric value.

Reply:
`encouraging_message`?

Reply:
Does it even need to be a string?
Could we make it a boolean? `personal_best`

What is it you're trying to help teach?

Reply:
What about making all the return values interpolated strings?